pom.xml

```diff
@@ -62,6 +67,31 @@
         <commons-io.version>2.1</commons-io.version>
         <node.version>v4.4.4</node.version>
         <npm.version>2.15.6</npm.version>
+        <!--Documentation Replacements-->
+        <branding>Alliance</branding>
+        <branding-lowercase>alliance</branding-lowercase>
+        <branding-expanded>Alliance</branding-expanded>
+        <ddf-branding>DDF</ddf-branding>
+        <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
+        <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <admin-console>Admin Console</admin-console>
+        <command-console>Command Console</command-console>
+        <!--Needed for documentation formatting-->
+        <at-symbol>@</at-symbol>
+        <variable-prefix>${</variable-prefix>
+        <image-width>500</image-width>
+        <!-- Individual application names -->
+        <ddf-admin>DDF Admin</ddf-admin>
+        <ddf-catalog>DDF Catalog</ddf-catalog>
```

Comment:
Eventually, yes, but I don't think that change has happened yet. I think the docs should just reflect what the user will see.